src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -58,7 +58,7 @@ def save_pretrained_wrapper(
             quantization_format: Optional[str] = None,
             save_compressed: bool = True,
             safe_serialization: bool = True,
-            skip_compression_stats: bool = False,
+            skip_sparsity_compression_stats: bool = False,
```

Comment:
```suggestion
            skip_sparsity_compression_stats: bool = True,
```

Since it's True in get_model_compressor as well